pkg/cmd/cli/cmd/importimage.go

```diff
@@ -147,7 +150,11 @@ func (o *ImportImageOptions) Run() error {
 	case err != nil:
 		return err
 	default:
-		fmt.Fprint(o.out, "The import completed successfully.\n\n")
+		if wasError(result) {
+			fmt.Fprint(o.errout, "The import did not complete successfully, please check below for details.\n\n")
```

Comment:
Don't say "successfully" if there was an error.   Also, we try to exit with "error: ", so I don't think this line would be correct.  It would be better to actually print out the error condition directly to the user water than print the describe.